### ISSUE TITLE
Esprima demo interface seems to be broken

### ISSUE THREAD

**Issue by goelayu** [jquery/esprima#2028]
### Steps to reproduce
1) Go to https://esprima.org/demo/parse.html#
2) The cursor location seems to be incorrect 

```js
esprima.parse('some code')
```

**Comment by ariya** [comment]
@goelayu Any information on the web browser and operating system used to reproduce this issue?

**Comment by nanyang24** [comment]
@ariya 
I made a gif to preview this issue: 


![Kapture 2020-07-17 at 15 21 12](https://user-images.githubusercontent.com/17287124/87759737-5902b300-c841-11ea-9e7b-9881c34cb844.gif)

**Comment by ariya** [comment]
The editor in the demo is [github.com/Microsoft/monaco-editor](https://github.com/Microsoft/monaco-editor), so likely it's related to that.
Can you try with the online [Monaco demo](https://microsoft.github.io/monaco-editor/playground.html)? Does it behave differently with another browser?

**Comment by nanyang24** [comment]
> The editor in the demo is [github.com/Microsoft/monaco-editor](https://github.com/Microsoft/monaco-editor), so likely it's related to that.
> Can you try with the online [Monaco demo](https://microsoft.github.io/monaco-editor/playground.html)? Does it behave differently with another browser?

Yes, I have tried again to make sure I only see the problem in the Esprima demo.
And I use Monaco Editor a lot in my work, maybe I can help you? 
But I don't actually see the original code for the demo in this repository, could you point me to it?

**Comment by ariya** [comment]
> And I use Monaco Editor a lot in my work, maybe I can help you?
> But I don't actually see the original code for the demo in this repository, could you point me to it?

Check the [gh-branches](https://github.com/jquery/esprima/tree/gh-pages). Maybe we just need an updated Monaco editor, and if you can help with that, it'll be fantastic!
